base_layer/core/tests/mempool.rs

```diff
@@ -754,7 +754,7 @@ fn service_request_timeout() {
 #[ignore = "Flaky test that needs to be fixed"]
```

Comment:
If the test is passing reliably now then you can remove this `#[ignore]` attribute so it can run on CI 👍

Reply:
done. thanks.